schema/deploy/tables/report_type.sql

```diff
@@ -3,47 +3,8 @@
 
```

Comment:
I haven't reviewed any post-prod db changes yet--is this how they normally turn out with sqitch?